package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "BarcodeScannerForPhoneGap",
+  "version": "3.0.1",
+  "description": "",
```

Comment:
Remove what you are not using or fill them in

Reply:
Done